packages/conf-diffutils/conf-diffutils.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
```

Comment:
I'm a bit surprised by the addition of this package. Why do you need it? Which distributions failed without it? Do you require a specific flavour of `diff` (GNU?) ?

Reply:
OK, it's clear I made too many changes in a single PR.  Again, you caught my oversight!

Camlp5's tests (which I want to enable) (specifically, the last test that is run) needs a diff that knows "-Bwiu".

Reply:
These options seem to be available on MacOS, FreeBSD and NetBSD so that sounds ok portability wise.
Opam already requires a posix `diff` command (see https://github.com/ocaml/opam/blob/master/src/client/opamInitDefaults.ml#L126) so I believe this package is not required

Reply:
Right, it was on a platform that ran everything in busybox, where I saw the error.  I can remove this prereq, and if it fails, put it back in.

Reply:
Ah! I forgot to check busybox :D I see! So that's why you marked Alpine as unavailable in the first place.
In that case could you advertise this package as installing GNU diff specifically and change the camlp5 package the following way?:
```
"conf-diffutils" { with-test & os-distribution = "alpine" }
```